lnrpc/invoicesrpc/addinvoice.go

```diff
@@ -53,70 +53,95 @@ type AddInvoiceConfig struct {
 	ChanDB *channeldb.DB
 }
 
+// AddInvoiceData contains the required data to create a new invoice.
```

Comment:
(atm) we already import channeldb (might change though, see comment on earlier commit), can we just use that struct directly here? Either way we need to convert the types _somewhere_

Reply:
The goal here is to make a clean object that specifies only data necessary to add the invoice. Not reuse a db struct with many fields that have no meaning during creating time.